src/main/java/teammates/storage/sqlentity/FeedbackResponse.java

```diff
@@ -0,0 +1,185 @@
+package teammates.storage.sqlentity;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.hibernate.annotations.UpdateTimestamp;
+
+import teammates.common.datatransfer.questions.FeedbackQuestionType;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Inheritance;
+import jakarta.persistence.InheritanceType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+/**
+ * Represents a Feedback Response.
+ */
+@Entity
+@Table(name = "FeedbackReponses")
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+public abstract class FeedbackResponse extends BaseEntity {
+    @Id
+    private UUID id;
+
+    @ManyToOne
+    @JoinColumn(name = "questionId")
+    private FeedbackQuestion feedbackQuestion;
+
+    @Column(nullable = false)
+    @Convert(converter = FeedbackParticipantTypeConverter.class)
+    private FeedbackQuestionType type;
+
+    @OneToMany(mappedBy = "feedbackResponse")
+    private List<FeedbackResponseComment> feedbackResponseComments = new ArrayList<>();
+
+    @Column(nullable = false)
+    private String giver;
+
+    @OneToOne
+    @JoinColumn(name = "giverSectionId")
+    private Section giverSection;
```

Comment:
This is not a OneToOne relationship. Multiple responses can have the same giverSection.

Reply:
right, changed to `ManyToOne`